src/next-app/components/EmailsField.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import styles from '../styles/components/EmailsField.module.css';
+import Chip from './Chip';
+import TextInput from './TextInput';
+
+type EmailsFieldProps = {
+  name: string;
+  placeholder?: string;
+  width?: string | number;
+  onChange?: (newVals: string) => void;
+  inputType?: string;
+
+  /** When any of the inputs lost focus */
+  onBlur?: () => void;
+
+  /** Makes the first input highlighted red */
+  error?: boolean;
+
+  /** Adds a red asterisk */
+  required?: boolean;
+};
+
+const EmailsField = (props: EmailsFieldProps) => {
```

Comment:
You should probably update this emails field component to make use of the email validation utility you made.

Reply:
I'll just merge it for now for the sake of getting it into main so everyone can work on their parts for the POC.